File: ET3ALEM_SERVER/BusinessEntities/Models/QuestionAttempt/LongAnswer.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace BusinessEntities.Models
{
    public class LongAnswer
    {
        public LongAnswer(int id, int longAnswerAttemptId, string answer)
        {
            Id = id;
            Answer = answer;
        }

        public int Id { get; set; }

        public LongAnswerAttempt LongAnswerAttempt { get; set; }
        public int LongAnswerAttemptId { get; set; }
        public string Answer { get; set; }
    }
}

File: ET3ALEM_CLIENT_WEB/src/app/question/Models/long-answer.ts
export class LongAnswer {
    Id: number;
    Answer: String;
    LongAnswerAttemptId: number;
    public constructor(Id: number, Answer: string, LongAnswerAttemptId: number) {
        this.Id = Id;
        this.Answer = Answer;
        this.LongAnswerAttemptId = LongAnswerAttemptId
    }
}